package.json

```diff
@@ -30,7 +30,10 @@
   "license": "MIT",
   "dependencies": {
     "@aws-sdk/client-dynamodb": "^3.341.0",
-    "@aws-sdk/util-dynamodb": "^3.341.0"
+    "@aws-sdk/client-lambda": "^3.362.0",
+    "@aws-sdk/util-dynamodb": "^3.362.0",
+    "eslint-config-airbnb-base": "^15.0.0",
```

Comment:
why airbnb? Is this what we use? Genuine question btw I have no idea. This package is srambiling Snyk tho so need to repair that.

Reply:
I added this as without it, eslint wasn't working was throwing an error like ESLint couldn't find the config "airbnb-base"

Reply:
Cool, just need to mess with the verisioning then to make snyk not hate you 😂

Reply:
should be a dev dependency though